### PR TITLE
Configure & run prettier

### PR DESCRIPTION
**Pull request by hanneskaeufler**
I noticed that my editor (which is configured to run prettier on save) and the checked in code sometimes disagreed.
Turns out, while prettier is a dev-dependency, it isn't enforced in any way and there were no easy to discover entrypoints in the package file.

This adds

* `npm run prettier-check`
* `npm run prettier-fix`

the first of which we could run in a github workflow if you agree.

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+**/.next/**
+**/_next/**
+.github/
+bazel-*
+node_modules
+pnpm-lock.yaml
```
Comment (hobofan) [inline .prettierignore:6]: This should also include the `out` directory, which is generated on `npm run build` (or `export`?). Without that `npm run prettier-check` fails.

**Review by hobofan — APPROVED** [review]
Adding it to the `deploy` workflow would be great!

File: .prettierignore (hunk 1 above)
Comment (hanneskaeufler) [reply .prettierignore:7]: Ah yeah, thx, done

**Comment by hanneskaeufler** [comment]
> 

I think I'd advise having this in a PR check. The deploy workflow seems like the wrong place to me. We can do that separately of course in another PR whichever way we end up going.

**Review by alexeagle — APPROVED** [review]
I think we should use pre-commit.com like the bazel-contrib/rules-template.

It's not very important whether there's CI enforcement for people installing it, IMO. We don't have that many contributions here.

**Pull request merged**